.github/ISSUE_TEMPLATE/deal-making-issues.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Deal Making Issues
+about: Create a report for help with deal making failures.
+title: "[Deal Making Issue]"
+labels: hint/needs-triaging, area/markets
+assignees: ''
+
+---
+
+Please provide all the information requested here to help us troubleshoot "deal making failed" issues.
+If the information requested is missing, we will probably have to just ask you to provide it anyway,
+before we can help debug.
+
+**Basic Information**
+Including information like, Are you the client or the miner? Is this a storage deal or a retrieval deal? Is it an offline deal?
+
+**Describe the problem**
+
+A brief description of the problem you encountered while deal making. 
+
+**Version**
+
+The output of `lotus --version`.
+
+**Setup**
+
+You miner(if applicable) and daemon setup.
+
+**Commands**
+
+Commands you ran.
```

Comment:
I'd use the 'To Reproduce' from bug-report.md, you don't have to run commands to make a deal

Reply:
updated